client/src/managers/userProfileManager.js

```diff
@@ -7,3 +7,7 @@ export const getProfiles = () => {
 export const getProfile = (id) => {
```

Comment:
Consider changing to `getProfileById()` to be in line with project naming convention